### PR TITLE
fixes 538 adding the isVisible function

### PR DESCRIPTION
**Pull request by jsanz**
I hope everything is fine :smile:

### CODE REVIEW

**Comment by jsanz** [comment]
Should I also document this, right?

File: test/spec/geo/sublayer.spec.js [hunk 1]
```diff
@@ -141,6 +141,17 @@ describe('Sublayers', function() {
       })
     });
 
+    describe('.isVisible', function() {
```
Comment (alonsogarciapablo) [inline test/spec/geo/sublayer.spec.js]: What about this:

```
it('should return true if sublayer is visible', function() {
  sublayer.set('hidden', undefined);

  expect(sublayer.isVisible()).toBeTruthy();

  sublayer.set('hidden', false);

  expect(sublayer.isVisible()).toBeTruthy();
})

it('should return false if sublayer is hidden', function() {
  sublayer.set('hidden', true);

  expect(sublayer.isVisible()).toBeFalsy();
})

```

Comment (jsanz) [inline test/spec/geo/sublayer.spec.js]: Better, thanks!

File: doc/API.md [hunk 2]
```diff
@@ -653,6 +653,10 @@ Removes the sublayer from the layer temporarily. The layer is refreshed after ca
 
 Toggles the visibility of the sublayer and returns a boolean that indicates the new status (true if the sublayer is visible, false if it is hidden)
 
+#### sublayer.isVisible()
+
+Gets the visibility of the sublayer.
```
Comment (alonsogarciapablo) [inline doc/API.md]: :+1: for updating the docs.

I'd be a bit more explicit about what this method returns. For example:
```
It returns `true` if the sublayer is visible.
```

File: test/spec/geo/sublayer.spec.js (hunk 1 above)
Comment (jsanz) [inline test/spec/geo/sublayer.spec.js]: Weird this works to test the falsy case

```javascript
        sublayer.hide();
        expect(sublayer.isVisible()).toBeFalsy();
```

but this doesn't

```javascript
        sublayer.set('hidden', true);
        expect(sublayer.isVisible()).toBeFalsy();
```

It's OK?

Comment (alonsogarciapablo) [inline test/spec/geo/sublayer.spec.js]: Sorry, perhaps it's:

```
sublayer.set({
  hidden: true
});
```
That's what `.hide` does internally.

Comment (jsanz) [inline test/spec/geo/sublayer.spec.js]: Thanks @alonsogarciapablo I think now it's more complete and working both tests

**Comment by alonsogarciapablo** [comment]
retest this please!

**Comment by jsanz** [comment]
Sorry :disappointed:, now it's passing the tests I think

**Comment by alonsogarciapablo** [comment]
:+1: Good job!

**Pull request merged**